rotkehlchen/tests/unit/test_evm_contracts.py

```diff
@@ -1,9 +1,40 @@
 import json
+from unittest.mock import patch
 
+import pytest
 from eth_utils import is_checksum_address
 
+from rotkehlchen.chain.ethereum.node_inquirer import EthereumInquirer
+from rotkehlchen.chain.evm.contracts import EvmContract
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+from rotkehlchen.tests.utils.factories import make_evm_address
+from rotkehlchen.tests.utils.mock import MockResponse
 from rotkehlchen.types import ChainID
 
+TEST_ABI_NAME = 'SOME_ABI_THAT_EXISTS_ON_REMOTE'
+
+TEST_ABI = [{
+    'constant': True,
+    'inputs': [],
+    'name': 'name',
+    'outputs': [{
+        'name': '',
+        'type': 'string',
+    }],
+    'payable': False,
+    'type': 'function',
+}]
+
+TEST_CONTRACT_NAME = 'SOME_CONTRACT_THAT_EXISTS_ON_REMOTE'
+
```

Comment:
Do we leave this space between constants? I believe we don't in other places of the code

```suggestion
```

Reply:
Okay